src/controllers/search.js

```diff
@@ -0,0 +1,20 @@
+const { getPropsalsbyValue } = require('../database/queries/getData');
+
+exports.search = (req, res) => {
+  const { inputvalue } = { ...req.body };
```

Comment:
@Fatmasiam I think a GET request typically won't have a `body` - I think you might need to instead use a query parameter for the browser to pass an input value, or turn this into a POST request.